modules/battenberg/1.0/envs/battenberg-1.0.yaml

```diff
@@ -0,0 +1,224 @@
+name: /home/rmorin/rrDLBCL_trios/.snakemake/conda/2f3a0b06
```

Comment:
I don't see a reference of this environment

Reply:
I'm a little worried the other one isn't complete. Can you test that the one used in the module works in a fresh environment (i.e. not in GAMBL)? If so, I'll delete this one.